### DISCUSSION TITLE
How do I call CLI commands while in development?

### DISCUSSION THREAD

**Discussion by ChristianPavilonis** [tauri-apps/tauri#6150] [Q&A] [answered]
The [docs](https://tauri.app/v1/guides/features/cli) explain how to define commands, but how can I call them while running `tauri dev`?

I'm trying do do something like `my-app /path/to/directory` to open a file.

**Comment by FabianLars** [comment, marked as answer]
`npm run tauri dev -- -- -- /path/to/directory` should work.
If you use cargo/pnpm/yarn instead of npm you need to remove one `--` => `cargo tauri dev -- -- /path/to/directory`
